style.css

```diff
@@ -46,18 +46,68 @@ body {
   padding-left: 24px;
 }
 
-nav {
+.hide {
+  display: none;
+}
+
+.navbar {
   display: flex;
-  flex: 1;
+  width: 100%;
+  min-height: 70px;
   justify-content: right;
   font-size: 15px;
   font-weight: 500;
   color: #344563;
-  padding-right: 24px;
+  padding-right: 0 24px;
 }
 
-i {
-  color: #6070ff;
+.nav-link {
+  color: white;
+  font-family: "Poppins", sans-serif;
+  font-size: 30px;
+}
+
+.hamburger {
+  display: block;
+  margin-right: 20px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 5px;
+  margin: 5px auto;
+  transition: all 0.3s ease-in-out;
+  background-color: #808080;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.nav-menu {
+  left: 100%;
+  top: 70px;
+  flex-direction: column;
+  background-color: #6070ff;
+  width: 100%;
+  text-align: start;
+  transition: 0.3s;
+  min-height: 700px;
+  z-index: 10;
+  opacity: 95%;
```

Comment:
- [ ] Good job so far by implementing the mobile menu, now it's functional but, to make it accepted by the standard and also presentable you have to fit exactly the design provide by the original design from Figma.

| Your design | Figma design |
| --------------| --------------- |
| ![image](https://user-images.githubusercontent.com/71874425/168111476-88b79581-6ced-4b2c-ab00-56be37dc06fd.png)| ![image](https://user-images.githubusercontent.com/71874425/168109251-2389195b-01eb-41a7-8ae5-833ceccf4af8.png)|

- Kindly, make sure that you fit exactly this template provided by the Figma design. background-color, font, size, and color should be the same. Let's rock, fix this issue 😃